storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfig.java

```diff
@@ -114,6 +120,14 @@ public static ConfigDef configDef() {
                 ConfigDef.NO_DEFAULT_VALUE,
                 ConfigDef.Importance.MEDIUM,
                 S3_REGION_DOC)
+            .define(
+                S3_STORAGE_CLASS_CONFIG,
+                ConfigDef.Type.STRING,
+                S3_STORAGE_CLASS_DEFAULT,
+                ConfigDef.ValidString.in(StorageClass.knownValues()
+                        .stream().map(Object::toString).toArray(String[]::new)),
+                ConfigDef.Importance.MEDIUM,
```

Comment:
maybe `LOW` as default covers most use-cases?

Reply:
done. I think MEDIUM is also ok due to it is more important when comparing with others. thus, when we consider that maybe some other don't care about it. so I agree to change it to low. thanks